R/dummy_functions.R

```diff
@@ -13,11 +19,12 @@
 #'   shinyApp(app$ui, app$server)
 #' }
 #' @export
-example_module <- function(label = "example teal module", datanames = "all", transformers = list()) {
+example_module <- function(label = "example teal module", datanames = "all", transformators = list(),
+                           decorators = teal_transform_module()) {
```

Comment:
either all args in single line or each arg in a new line
```suggestion
example_module <- function(label = "example teal module", 
                           datanames = "all", 
                           transformators = list(),
                           decorators = teal_transform_module()) {
```